Makefile

```diff
@@ -17,7 +17,7 @@ venv:
 
 test-integration:
 	podman build --no-cache -t cachi2-${USER} .
-	CACHI2_IMAGE=localhost/cachi2-${USER}:latest tox -e integration
+	CACHI2_UPDATE_TEST_DATA=false CACHI2_IMAGE=localhost/cachi2-${USER}:latest tox -e integration
```

Comment:
Could we have a new command for generating test data? It would basically be a copy-paste of this one just with `CACHI2_UPDATE_TEST_DATA=true`

And the `test-integration` command doesn't need to set CACHI2_UPDATE_TEST_DATA, unset should count as false